.github/workflows/docs.yaml

```diff
@@ -0,0 +1,26 @@
+name: GitHub pages
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+      - run: npm i
+      - run: npm install jsdoc ink-docstrap tsd-jsdoc
+      - name: Build
+        uses: andstor/jsdoc-action@v1
```

Comment:
This action doesn't play nice with newer syntax, probably just manually call `npm run build:docs`